CHANGELOG.md

```diff
@@ -5,6 +5,70 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.1] - Upcoming release
+
+### Dependencies
+
+| Package                   | Version |
+| ------------------------- | ------- |
+| @chainlink/contracts-ccip | 1.4.0   |
+| @chainlink/contracts      | 1.1.1   |
+
+### Services
+
+- [x] Chainlink CCIP
+- [x] Chainlink Data Feeds
+- [ ] Chainlink VRF 2
+- [ ] Chainlink VRF 2.5
+
+### Added
+
+- Support for Chainlink Data Feeds by adding `MockV3Aggregator.sol` and `MockOffchainAggregator.sol` mock contracts
+- Showcase tests for testing in a forking actual networks environment
+
+## [0.2.1-beta] - 26 June 2024
+
+### Dependencies
+
+| Package                   | Version |
+| ------------------------- | ------- |
+| @chainlink/contracts-ccip | 1.4.0   |
+| @chainlink/contracts      | 1.1.1   |
+
+### Services
+
+- [x] Chainlink CCIP
+- [x] Chainlink Data Feeds
+- [ ] Chainlink VRF 2
+- [ ] Chainlink VRF 2.5
+
+### Fixed
+
+- Bug in propose & confirm aggregator flow that could lead to aggregator being set to `address(0)`
```

Comment:
is it worth linking to the file or commit in which the bug was fixed for future tracing?